### ISSUE TITLE
[1.9.24] iPhone 6 / 10.0.1 - Black screen on preview screen - Create lomotif with camera roll image

### ISSUE THREAD

**Issue by hiteshbheda** [Lomotif/lomotif-ios-issues#727]
Frequency: Many times (7 / 10)

Action: 
1. Capture an image from iOS device camera (HD image)
2. Launch Lomotif app
2. Tap on "START" button
3. Select recently captured image from Camera Roll folder
4. Don't add any song and create lomotif
5. On preview screen, check preview screen is Black or not

Expected result: On preview screen, lomotif should play properly without any black frames

Actual result: Black screen displays on Preview screen

**IMPORTANT NOTE: If user find Black frames on preview screen and export the same lomotif -> Open in Camera app, It plays perfectly without Black frames**

**Comment by testershub** [comment]
Similar to #705 

@xiuie I can reproduce this issue for build [1.9.24] on iPhone 6S / 10.0.1.
I can't reproduce this issue on iPhone 6 Plus / 10.0.1 and iPad Mini 3 / 9.3.4.

**Comment by hiteshbheda** [comment]
@xiuie @testershub Thanks!

But this issue is not exactly similar to #705. 
Because in #705 in Exported lomotif also Black frames were appearing but in this issue for build [1.9.24] exported preview / video plays perfetly only lomotif created screen has Black frames available.

Thanks @testershub

**Comment by xiuie** [comment]
@hiteshbheda I dont understand how it's different too. This means to say (between the two reports), that if they follow till step 5, and they export, sometimes its black and sometimes it's not?

**Comment by hiteshbheda** [comment]
@xiuie Yes, you are right. Till step no. 5 it is reproducible but after exporting from camera app it is not reproducible .

Exported lomotif plays perfectly without black frames
